### PR TITLE
chanching UP, and altitude

### PR DESCRIPTION
**Pull request by darkbushido**
using SHIP:UP so the navball makes sense

going with altitude instead of apoapsis for target pitch

### CODE REVIEW

File: dd_launch [hunk 1]
```diff
@@ -36,14 +36,13 @@ print "Liftoff!".
 wait until verticalSpeed > 10.
 
 local lock lookAt to heading(launchHeading, 90).
-local lock lookUp to heading(launchHeading, -45).
-lock steering to lookdirup(lookAt:vector, lookUp:vector).
+lock steering to lookdirup(lookAt:vector, SHIP:UP:vector).
```
Comment (jberkman) [inline dd_launch:39]: 1. generally when contributing to another codebase, you should try to match their coding style - this should be `ship:up:vector`.
2. could you explain a little more what the problem you're seeing is? how does the navball not make sense?

File: dd_launch [hunk 2]
```diff
@@ -36,14 +36,13 @@ print "Liftoff!".
...
 
 if body:atm:exists {
   wait until verticalSpeed > 50.
 
   print "Initiating gravity turn.".
-  lock targetPitch to 90 - 90 * min(1, apoapsis / (body:atm:height - 10000)).
+  lock targetPitch to 90 - 90 * min(1, altitude / (body:atm:height - 10000)).
```
Comment (jberkman) [inline dd_launch:45]: no, i really mean to target apo here.  i've tried altitude (and averaging altitude and apo).

the intent is to level out while still inside very thin atmosphere. this lets us take advantage of the oberth effect a little bit, and keep our circularization burn low. it also enables coasting to apo without having to reboost.

i know it's not a typical ksp launch or the most efficient, but i've enjoyed using it recently.

**Comment by darkbushido** [comment]
My apologizes, I didn't mean for the altitude change to get in that. 

ship:up:vector
This is a matter of preference, if you were to put a cockpit on the craft would you expect the pilot to look at the ground or out to space.

**Comment by jberkman** [comment]
it's designed to keep the ship's azimuth/dorsal pointing towards the surface, like the shuttle and Saturn V (in order for the horizon to be visible during launch). if that's not what you're seeing, would you mind attaching a craft file (or a link to kerbalx)?

hth

**Pull request closed without merging**